openldap/2.4.44/enable_monitor.sh

```diff
@@ -0,0 +1,26 @@
+log-helper info "Configuring monitor backend..."
+cat <<EOF | ldapmodify -Y EXTERNAL -H ldapi:///
+dn: cn=module{0},cn=config
+changetype: modify
+add: olcModuleLoad
+olcModuleLoad: back_monitor.la
+EOF
+
```

Comment:
Could you add log lines just so we understand better what are these steps. It won't hurt :D